code/game/objects/items/implants/implant.dm

```diff
@@ -1,12 +1,20 @@
+/**
+ * Parent class for all implants
+ */
 /obj/item/implant
 	name = "implant"
 	icon = 'icons/obj/implants.dmi'
 	icon_state = "generic" //Shows up as the action button icon
 	actions_types = list(/datum/action/item_action/hands_free/activate)
-	var/activated = TRUE //1 for implant types that can be activated, 0 for ones that are "always on" like mindshield implants
+	///true for implant types that can be activated, false for ones that are "always on" like mindshield implants
+	var/activated = TRUE
+	///the mob that's implanted with this
 	var/mob/living/imp_in = null
+	///implant color, used for visuals
 	var/implant_color = "b"
+	///if false, upon implantation of a duplicate implant, an attempt to combine the new implant's uses with the old one's uses will be made, deleting the new implant if succesful or stopping the implantation if not
 	var/allow_multiple = FALSE
+	///uses
 	var/uses = -1
```

Comment:
I am unclear if this documention is actually worth anything.

Reply:
improved.